migrations/versions/e360c56bcf8c_.py

```diff
@@ -0,0 +1,83 @@
+"""empty message
```

Comment:
Please add a nice message, that would be displayed in 

    pi-manage db history

like: *Add constraint for smsoptions and set default type to "option"*

Reply:
ok